src/js/update-sites.js

```diff
@@ -20,26 +22,29 @@ let vm = new Vue({
     packages: [],
     codes: [],
     sites: [],
-    siteQuery: []
+    siteQuery: '?where={"code.package":"588b90ab36447521c5af51e4"}'
   },
   methods: {
     getSites(siteQuery) {
       let that = this;
-      let endpoint = '/sites';
+      let endpoint = 'sites';
+      console.log(packageBaseURL+endpoint+siteQuery);
 
-      atlasRequest(baseURL, endpoint, query = siteQuery, method = 'GET', body = null, etag = null).then(data => that.sites = data);
+      atlasRequest(packageBaseURL, endpoint, query = siteQuery, method = 'GET', body = null, etag = null).then(data => that.sites = data);
+      setTimeout(remDup, 50);
```

Comment:
I would filter the list of sites on line 33 in the thenable before assigning to `that.sites`.